Src/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -72,12 +78,20 @@ public override AuthorizationCodeRequestUrl CreateAuthorizationCodeRequest(strin
             await DeleteTokenAsync(userId, taskCancellationToken);
         }
 
+        public override bool ShouldForceTokenRetrieval() 
+        {
+            return includeGrantedScopes.HasValue && includeGrantedScopes.Value == true;

```

Comment:
oops, actually you don't need the "== true" as well so:
return includeGrantedScopes.HasValue && includeGrantedScopes.Value;